Clinical-Genomics/cgweb
# Show customer when searching for family

How to test:
0. install branch on clinical-db stage
1. login into OP-stage
2. "New Order"
3. Enter order name "#123456", Select type "Sequencing and Scout", select cust000
4. Save order
5. Enter test in family name
Expected outcome:
Several families are found and after the name of each, the customer should be displayed within parentheses like when you search for samples
